docker/compose-services.yml

```diff
@@ -17,3 +17,20 @@ services:
       - 9000:9000
       - 2500:2500
       - 1100:1100
+
+  minio:
+    image: minio/minio
+    container_name: s3
+    ports:
+      - 9002:9002
+      - 9001:9001
+    volumes:
+      - minio:/data
+    environment:
+      MINIO_ROOT_USER: documenso
+      MINIO_ROOT_PASSWORD: password
+    entrypoint: sh
+    command: -c 'mkdir -p /data/documenso && minio server /data --console-address ":9001" --address ":9002"'
```

Comment:
1. is the `entrypoint` field necessary?
2. is this part `-c 'mkdir -p /data/documenso && minio` from the `command` necessary? doesn't it work without it?

I'm asking because I don't have much Docker experience, so I'm trying to understand. :-D

Reply:
Yes and yes, `entrypoint` allows me to run the command after the container has been initiated. The command allows me to create a bucket with name `documenso` else the user would have to create the bucket manually which is easier to miss and you get a lot of errors if you don't have the bucket available